### PR TITLE
ci: building pdfs silently

### PR DESCRIPTION
**Pull request by shuzhouliu**
By purposing this change, packages install and precompile commands will be silenced, latex compilation log info will also be silenced. But errors & warnings in latex have not been silenced.

There will possibly be a long period of silence in build, in this case wait. (The only difference between the silence and the old one is logs not necessarily needed.)

### CODE REVIEW

**Comment by shuzhouliu** [comment]
https://github.com/shuzhouliu/OI-wiki/actions/runs/2666801898 

ran on this repo before submit, you can preview status here

**Comment by Enter-tainer** [comment]
可以直接在原来的yml里面修改吗？感觉已经不需要之前的pdf build了🤔

**Comment by shuzhouliu** [comment]
> 可以直接在原来的yml里面修改吗？感觉已经不需要之前的pdf build了🤔

done

File: .github/workflows/build-pdf.yml [hunk 1]
```diff
@@ -16,43 +16,42 @@ jobs:
...
-            git clone https://github.com/OI-wiki/OI-Wiki-export.git --depth=1
+            apk add unzip > /dev/null
+            apk add wget > /dev/null
+            apk add curl > /dev/null
+            apk add coreutils > /dev/null
+            apk add nodejs > /dev/null
+            apk add npm > /dev/null
+            apk add imagemagick > /dev/null
+            apk add libwebp > /dev/null
+            apk add librsvg > /dev/null
```
Comment (Enter-tainer) [inline .github/workflows/build-pdf.yml:29]: 思考 这些log要不要干掉呢

Comment (Enter-tainer) [reply .github/workflows/build-pdf.yml:29]: 感觉之前的log中，绝大部分是LaTeX打印出来的。前面这些应该还好？

...不过一般这块也不会出问题(?)，删掉应该也问题不大，，

Comment (shuzhouliu) [reply .github/workflows/build-pdf.yml:29]: > 感觉之前的log中，绝大部分是LaTeX打印出来的。前面这些应该还好？
> 
> ...不过一般这块也不会出问题(?)，删掉应该也问题不大，，

如果有error 应该会报的吧

Comment (shuzhouliu) [reply .github/workflows/build-pdf.yml:29]: 如果没有error 看这些也没啥意思 😆

**Comment by shuzhouliu** [comment]
image

tests for ci `Build LaTeX document / build (push)` seem to pass, I canceled `Build LaTeX document / build (pull_request)` because it was unneccesary.

**Review by Enter-tainer — DISMISSED** [review]

**Comment by shuzhouliu** [comment]
can't optimize so rollback

**Comment by shuzhouliu** [comment]
latex done
works

**Comment by shuzhouliu** [comment]
> 你merge一下master吧，让ci的红叉消掉（

done

**Review by Enter-tainer — APPROVED** [review]

**Comment by shuzhouliu** [comment]
(test if bot working

**Comment by shuzhouliu** [comment]
有write access的人review下这个吗（

**Review by CoelacanthusHex — APPROVED** [review]

**Pull request merged**